tests/bud.bats

```diff
@@ -1140,3 +1140,19 @@ load helpers
   echo "$output"
   test "${#lines[@]}" -eq 0
 }
+
+@test "bud copy to symlink" {
+  target=alpine-image
+  ctr=alpine-ctr
+  run buildah --debug=false bud --signature-policy ${TESTSDIR}/policy.json -t ${target} ${TESTSDIR}/bud/dest-symlink
```

Comment:
`run_buildah` (with underscore) gives you exit code checking, timeout, and much better logging in case of error.

Reply:
fixed